htdocs/product/list.php

```diff
@@ -1643,7 +1643,9 @@
 				if ($obj->seuil_stock_alerte != '' && $product_static->stock_reel < (float) $obj->seuil_stock_alerte) {
 					print img_warning($langs->trans("StockLowerThanLimit", $obj->seuil_stock_alerte)).' ';
 				}
-				print price2num($product_static->stock_reel, 'MS');
+				$decimals = !isset($conf->global->MAIN_MAX_DECIMALS_STOCK) ? 5 : $conf->global->MAIN_MAX_DECIMALS_STOCK;
```

Comment:
Using the "MS" of price2num is already the "shared code" to round an amount of stock.
Why not just adding price to format amount into locale format.

print price(price2num($product_static->stock_reel, 'MS'));